metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "theforeman/foreman_proxy",
-      "version_requirement": ">= 13.0.0 < 18.0.0"
+      "version_requirement": ">= 16.1.0 < 17.0.0"
```

Comment:
Release PR @ https://github.com/theforeman/puppet-foreman_proxy/pull/644

Reply:
I've updated the floor to 17.1.0 and the ceiling back to 18